ui/src/components/Operations/Operations.tsx

```diff
@@ -41,64 +41,22 @@ export function Operations(props: OperationsOwnProps) {
             {metaInProgress ? (
                 <Button loading type="link" className={styles.loading} />
             ) : (
-                currentOperations.map((item: Operation | OperationGroup, index) => {
+                currentOperations.map((item: Operation | OperationGroup) => {
                     if ((item as OperationGroup).actions) {
                         const group = item as OperationGroup
-                        if (group.type && hiddenGroups && hiddenGroups.includes(group.type)) {
-                            return null
-                        }
-
-                        let groupIsEmpty = true
-                        const moreOperations = (
-                            // todo styles
-                            <Menu>
-                                {group.actions.map(operation => {
-                                    if (removeRecordOperation && operation.scope === 'record') {
-                                        return null
-                                    }
-
-                                    groupIsEmpty = false
-                                    return (
-                                        <Menu.Item
-                                            key={operation.type}
-                                            className={styles.subOperation}
-                                            onClick={createClickHandler(operation)}
-                                        >
-                                            {operation.icon && <Icon type={operation.icon} />}
-                                            {operation.text}
-                                        </Menu.Item>
-                                    )
-                                })}
-                            </Menu>
-                        )
-
-                        if (groupIsEmpty) {
-                            return null
-                        }
-
-                        const trigger = (
-                            <OperationButton key={item.text}>
-                                <Icon type="file-add" />
-                                {item.text}
-                            </OperationButton>
-                        )
-
-                        return group.actions.length ? (
-                            <Dropdown
-                                trigger={['click']}
-                                overlay={moreOperations}
-                                key={index}
-                                getPopupContainer={element => element.parentElement as HTMLElement}
-                            >
-                                {trigger}
-                            </Dropdown>
-                        ) : (
-                            trigger
+                        return (
+                            <OperationsGroup
+                                key={group.type}
+                                group={group}
+                                hiddenGroups={hiddenGroups}
```

Comment:
`hiddenGroups` are deprecated and shouldn't be present in the demo or archetype

Reply:
It's strange IDE doesn't highlight that so I haven't noticed that. I'll remove it, thank you